src/devices/ledvance.ts

```diff
@@ -332,6 +332,15 @@ const definitions: DefinitionWithExtend[] = [
         description: 'SMART+ Outdoor Plug EU',
         extend: [ledvanceOnOff({powerOnBehavior: false})],
     },
+    {
+        zigbeeModel: ['PLUG COMPACT EU EM T'],
+        model: 'PLUG COMPACT EU EM T',
```

Comment:
Use the model number written on the device here

Reply:
Model is 4099854293276